### PR TITLE
Removes the elite security skirt

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Done on popular demand
:cl:
 * rscdel: The elite security skirt is no longer in the game.

### CODE REVIEW

**Comment by plosky1** [comment]
this is not ok

**Comment by APCUnlocked** [comment]
This man is singlehandedly saving /vg/ security.

**Comment by nuklearcellphoneg** [comment]
Finally, he codes something worth a damn

**Comment by DrSnips** [comment]
The tranny metaclub will remember this

**Comment by cloutcode** [comment]
You look at all the people upvoting, these people have been forced to wear the skirt

**Comment by PrimeDSS13** [comment]
Finally, the security buff we've been waiting for.

**Review by Wilchenx — APPROVED** [review]

**Comment by ghost** [comment]
Yeah remove it

**Comment by SplergeBucket** [comment]
Fantastic PR, cutting edge stuff.

File: code/game/objects/items/trader.dm [hunk 1]
```diff
@@ -350,6 +350,7 @@
 			last_donut = world.timeofday
 
 //Security Skirt spritework is coutesy of TG, CC BY-SA 3.0 license
+/*
```
Comment (jknpj) [inline code/game/objects/items/trader.dm]: Do not comment like this, it fucks up conflict solving. Also why not just remove the code? And the icons?

Comment (boy2mantwicethefam) [reply code/game/objects/items/trader.dm]: It messes up conflict solving?
2) I am lazy and our sins must be exposed, history repeats itself and all that

**Comment by SonixApache** [comment]
epic, truly epic

notice the complete lack of reason anywhere; meme PR for the sake of making meme PRs
complete with i-don't-play-the-game-but-still-vote epic upboats

**Comment by Intigracy** [comment]
I think the problem is the paracet, not the clothing itself

**Comment by RFultra** [comment]
All memes aside, grudgecoding and feature loss is bad. Stop it. Get some help.

**Comment by SplergeBucket** [comment]
> 
> 
> epic, truly epic

ok the main member of the sec skirt society is on board with the PR
push it through now

File: code/game/objects/items/trader.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/game/objects/items/trader.dm]: Anyway I am going to remove it from code BRB

**Comment by nuklearcellphoneg** [comment]
Hurry the hell up and merge it

**Comment by ghost** [comment]
SSSSD
i thought we werent tg???

**Comment by ghost** [comment]
But how are we going to keep our incredible FemSec roleplay up ?

**Comment by N3X15** [comment]
Hide it in the HoS' locker.

**Comment by RealestEstate** [comment]
grudge coding and feature removal is bad unless it removes things I don’t like. very good.

**Comment by Kammerjunk** [comment]
quick map the skirt in
then whomsoever takes up the removal PR shall deal with DMM

**Comment by juanototo2** [comment]
Remove the security skirt and the usual suspects will flock to maid outfits instead. The only effective solution is either administrative or instahusking them the second they touch an AutoDrobe. I'll still support this anyways.

**Comment by Lokiamis** [comment]
""""""Feature""""""

**Comment by ArchSunder** [comment]
image

the coomer manifest

**Comment by mentgah** [comment]
>more meme PRs

**Comment by yclatious** [comment]
Based&Redpilled, Hoes Mad.

Dab on the waifus, dab on the fags, tear up that secskirt into rags, greytide greytide greytide greytide.

**Comment by Tokanova** [comment]
fuck u

**Comment by yclatious** [comment]
HOES MAD x24

**Comment by Kurfursten** [comment]
Needs compensation for traders, bangerboy is already pretty meh.

Make something new for the crate, until then this is :thumbsdown:

**Comment by ComradeDispenser** [comment]
Great, remove the one and only item that gives traders any business.

**Comment by ghost** [comment]
imagine being such an autistic neckbeard that you get triggered by a skirt existing in your janky ass spessman simulator

**Comment by yclatious** [comment]
Imagine being such a fag neckbeard you get triggered a skirt is removed from your janky ass spessman simulator

Feckin furries.

**Comment by SonixApache** [comment]
imagine being such an autistic neckbeard that you get triggered by someone calling you out over getting triggered by a skirt existing in your janky ass spessman simulator

**Comment by yclatious** [comment]
Stop projecting Sonix

**Comment by ShiftyRail** [comment]
The only PR that would fix what this is attempting to fix is a rollback to 2016 with a note to all of our past selves
But I give this A+ for effort

And to make this less of a meme comment : /vg/station, not /trash/station OK

**Comment by Kammerjunk** [comment]
god why did I ever leave a comment here
image
barry is trying to be an epic meme master like always, yclat makes me cringe whenever he types, and the existence of this item is so irrelevant that you'd be playing the *exact* same game regardless of whether the PR is merged or closed
that this PR has 42 "thumbs" votes is proof that bikeshedding is not a lost art

**Comment by ShiftyRail** [comment]
Kammer, in the event you didn't do this already, you can turn off notifications

**Comment by Exxion** [comment]
It's a joke item, and a pretty funny one at that
But really who gives a shit either way

**Comment by bathosbathtime** [comment]
We should have the skirt replace the regular security uniforms completely.  Including those blasted white uniforms.  If we move far enough into the extreme, it will overflow error back into perfection.
Let the vox traders have the old uniforms if it is mission critical the smelly birds have their toys to trade.

**Comment by 9percents** [comment]
this PR and discussion are moot
this is just a skirt

**Comment by Exxion** [comment]
It is with a heavy heart I must add this label to a PR regarding a joke item

**Comment by ghost** [comment]
I can't remember, does the sec skirt have a fat version of the sprite? If it doesn't then it needs to have one that is incredibly awful made for it.

**Comment by tuypo1** [comment]
In my probably dozens of hours playing warden I have never seen the sort of things this PR is meant to be stopping actually happen, I am pretty sure it's all just shitposting people have taken too seriously

**Comment by Intigracy** [comment]
Barry is a mistake

**Comment by BurntDevil** [comment]
I am spriting a gender flag male version of this to correct the issue. Removal will be unnecessary.

**Comment by Intigracy** [comment]
Crossdressing is the tranny metaclub's god given right

**Comment by Kurfursten** [comment]
It's been two weeks, the support isn't there

**Pull request closed without merging**